distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -1853,6 +1853,7 @@ The variable version is longer but will not need to be changed if the system hos
 A banner is displayed on the admin console notifying "The system is insecure because default configuration values are in use."
 
 A detailed view is available of the properties to update.
+====
```

Comment:
This line is duplicating the intent of line 1858 and breaking the generation of the docs from this point forward :(